overlord/ifacestate/schemas/schemas.go

```diff
@@ -0,0 +1,44 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
I would call this package "schema", not schemas

Reply:
Fixed now